### PR TITLE
ci: support prerelease

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [x] Build or CI related changes
- [ ] Documentation content changes

### CODE REVIEW

File: scripts/shared/bump-version.ts [hunk 1]
```diff
@@ -10,6 +12,8 @@ export function bumpVersion(version: string, mode: TuiReleaseMode): string {
             return `${major}.${++minor}.0`;
         case 'patch':
             return `${major}.${minor}.${++patch}`;
+        case 'prerelease':
+            return `${alpha === -1 ? ++major : major}.0.0-${++alpha}`;
```
Comment (nsbarsukov) [inline scripts/shared/bump-version.ts:16]: Should we add word `alpha` in the version ?

For example, `3.0.0-alpha.3` or `3.0.0-beta.1` ?
Or `3.0.0-3` is enough  for our needs ?

**Review by nsbarsukov — APPROVED** [review]

File: scripts/shared/bump-version.ts (hunk 1 above)
Comment (splincode) [reply scripts/shared/bump-version.ts:16]: as you wish, so be it

File: scripts/shared/bump-version.ts [hunk 2]
```diff
@@ -1,7 +1,9 @@
 import {TuiReleaseMode} from './release-mode';
 
 export function bumpVersion(version: string, mode: TuiReleaseMode): string {
-    let [major, minor, patch] = version.split('.').map(value => Number(value));
+    let [major, minor, patch, , rc = -1] = version
+        .split(/[.-]/)
```
Comment (nsbarsukov) [inline scripts/shared/bump-version.ts:5]:
```ts
'3.0.0-rc.2'.split(/[.-]/) // ['3', '0', '0', 'rc', '2']
```

Comment (nsbarsukov) [reply scripts/shared/bump-version.ts:5]: opps, sorry

Comment (nsbarsukov) [reply scripts/shared/bump-version.ts:5]: missed two commas))

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**